refinery/data_set_manager/utils.py

```diff
@@ -734,14 +734,16 @@ def format_solr_response(solr_response):
     except KeyError:
         order_facet_fields = []
 
-    if solr_response_json.get('facets'):
+    if solr_response_json.get('facets') and include_facet_count != 'false':
         solr_response_json['facet_field_counts'] = create_facet_field_counts(
             solr_response_json.get('facets')
         )
-        del solr_response_json['facets']
     else:
         solr_response_json['facet_field_counts'] = {}
 
+    if solr_response_json.get('facets'):
```

Comment:
did you have a preference for leaving this up here as opposed to down with the other chunk of clean-up code at 744?

Reply:
I'm okay with it being here, but if you feel strongly about it I can relocate it.

Reply:
all good with me, just checking